### PR TITLE
[VEN-2548] Add Pendle weETH Market

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2548

### CODE REVIEW

File: multisig/simulations/ethereum/vip-023/index.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        parseUnits("1", 18),
+      );
+    });
+
+    it("should have 3 markets in liquid staked pool", async () => {
+      const poolVTokens = await comptroller.getAllMarkets();
+      expect(poolVTokens).to.have.lengthOf(4);
+    });
+
+    it("should add vweETH to the pool", async () => {
```
Comment (chechu) [inline multisig/simulations/ethereum/vip-023/index.ts:94]:
```suggestion
    it("should add vPTweETH to the pool", async () => {
```

File: multisig/proposals/sepolia/vip-023/index.ts
[the remark this answers is not in the document]
Comment (chechu) [reply multisig/proposals/sepolia/vip-023/index.ts:1]: The added multisig TX on sepolia to set the borrow cap to zero lgtm

**Comment by GitGuru7** [comment]
Normal VIP for mainnet, with a single command transferring 1 wei from/to the fast-track timelock is missing

File: multisig/simulations/ethereum/vip-023/index.ts [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    it("check ownership", async () => {
+      expect(await vPTweETHContract.owner()).to.equal(ethereum.GUARDIAN);
+    });
+
+    it("check supply", async () => {
+      const expectedSupply = parseUnits("1.79961879", 8);
+      expect(await vPTweETHContract.balanceOf(ethereum.VTREASURY)).to.equal(expectedSupply);
+    });
+
+    describe(`check risk parameters`, () => {
```
Comment (GitGuru7) [inline multisig/simulations/ethereum/vip-023/index.ts:108]: I think we can add a test to check reduceReservesBlockDelta

File: multisig/proposals/ethereum/vip-023/index.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+const { ethereum } = NETWORK_ADDRESSES;
+
+export const PENDLE_ORACLE = "0xA1eA3cB0FeA73a6c53aB07CcC703Dc039D8EAFb4";
+export const PTweETH = "0x6ee2b5E19ECBa773a352E5B21415Dc419A700d1d";
+export const VTREASURY = "0xFD9B071168bC27DBE16406eC3Aba050Ce8Eb22FA";
+export const vPTweETH = "0x76697f8eaeA4bE01C678376aAb97498Ee8f80D5C";
+export const COMPTROLLER = "0xF522cd0360EF8c2FF48B648d53EA1717Ec0F3Ac3";
+export const INITIAL_SUPPLY = parseUnits("1.799618792392372642", 18);
+
+export const TWAP_DURATION = 1800;
```
Comment (GitGuru7) [inline multisig/proposals/ethereum/vip-023/index.ts:16]:
```suggestion
```

File: vips/vip-295/bscmainnet.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    [
+      {
+        target: NORMAL_TIMELOCK,
+        signature: "",
+        params: [],
+        value: "1",
+      },
+    ],
+    meta,
+    ProposalType.REGULAR,
```
Comment (GitGuru7) [inline vips/vip-295/bscmainnet.ts:26]: In ticket it is mentioned to use FAST_TRACK timelock.

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/305?chainId=56

**Pull request merged**